app/client/packages/design-system/widgets/src/components/DatePicker/src/DatePicker.tsx

```
@@ -0,0 +1,70 @@
+            isRequired={isRequired}
+            text={label}
+          />
+          <Group className={styles.inputWrapper}>
+            <DateInput
+              className={clsx(styles.input, getTypographyClassName("body"))}
+              data-date-input
+            >
+              {(segment) => <DateSegment segment={segment} />}
+            </DateInput>
+            <Button
+              color={Boolean(isLoading) ? "neutral" : "accent"}
```

Biome

[error] Avoid redundant `Boolean` call

It is not necessary to use `Boolean` call when a value will already be coerced to a boolean.
Unsafe fix: Remove redundant `Boolean` call


(lint/complexity/noExtraBooleanCast)